### DISCUSSION TITLE
I cant listen the change of input in livewire 3

### DISCUSSION THREAD

**Discussion by allyson-silva** [livewire/livewire#8082] [1. Help]
This is my class

`public function updatedSelectedValue()
    {
        // Handle the selected value here, e.g., update a model, etc.
        dd($this->selectedValue);
    }`

And this is my Blade

`<div style="text-align: center">
  <select wire:model="selectedValue"> 
    <option value="">Selecione uma opção</option>
      @foreach (range(1, 10) as $value) <option value="{{ $value }}">Opção {{ $value }}</option>
      @endforeach
  </select>

  @if ($errors->has('selectedValue'))
      <div class="error-message">{{ $errors->first('selectedValue') }}</div> @endif

  O valor selecionado é {{ $selectedValue }}
</div>`

**Comment by PhiloNL** [comment]
Hi @allyson-silva,

This should work: https://wirebox.app/b/ywdve
Make sure you use `wire:model.live` if you want the event to fire on every change instead of waiting for the form to be submitted.
